packages/config/config/devices/0x0438/4512757.json

```
@@ -0,0 +1,534 @@
+{
+	"manufacturer": "Namron",
+	"manufacturerId": "0x0438",
+	"label": "4512757",
+	"description": "16A zwave thermostat - White",
```

```suggestion
	"description": "16 A Thermostat - White",
```
No need to mention zwave here.

---

Corrected these throughout the file